book/src/skip-attach.md

```diff
@@ -1,6 +1,6 @@
 # Skip Kubernetes Attach and Detach
 
-**Status:** Alpha
+**Status:** Beta
```

Comment:
Add the following line below this:

The "Skip Kubernetes Attach and Detach" feature was introduced as alpha in Kubernetes v1.12. It was promoted to beta in Kubernetes 1.13.